src/Compilers/Core/MSBuildTaskTests/DotNetSdkTests.cs

```diff
@@ -407,7 +407,10 @@ public void TestDiscoverEditorConfigFiles()
             var editorConfigFile2 = subdir.CreateFile(".editorconfig").WriteAllText(@"[*.cs]
 some_prop = some_val");
             VerifyValues(
-                customProps: null,
+                customProps: @"
+<PropertyGroup>
+    <GenerateMSBuildEditorConfigFile>false</GenerateMSBuildEditorConfigFile>
```

Comment:
❔ Can you provide more context on this change (e.g. why it applies to this set of tests, and who/when it would apply to others)?

Reply:
The NET 6 SDK now automatically generates global editorconfig from MSBuild. Which breaks all these tests because they are not expecting an additional global config.

Reply:
I could have updated the tests to expect a generated global config, but then the tests would fail against older SDKs, which may not be a problem.

Reply:
this seems like the right fix to me, ca you add a comment calling this out?